eng/testing/ildescriptors/ILLink.Descriptors.Serialization.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
+  <assembly fullname="System.Private.DataContractSerialization">
+    <type fullname="System.Runtime.Serialization.KeyValue`2" />
```

Comment:
Are these still necessary on the latest main? @joperezr addressed the ILLink warnings in this assembly, so these might be fixed.

If they aren't fixed, we should open a bug to make sure we don't have a product bug here.

Reply:
DCS is annotated now but most of it was annotated as unsafe. It would be good to understand who actually needs this so that we can check why is the type not being preserved and to find out if it is expected or not